contracts/contracts/strategies/Generalized4626Strategy.sol

```diff
@@ -35,7 +35,7 @@ contract Generalized4626Strategy is InitializableAbstractStrategy {
      * @param _asset Address of asset to deposit
      * @param _amount Amount of asset to deposit
      */
-    function _deposit(address _asset, uint256 _amount) internal {
+    function _deposit(address _asset, uint256 _amount) internal virtual {
```

Comment:
Since this contract is extended for the first time, lets add some storage slot padding to it. Just in case we ever need to alter it in the future.

Reply:
Agree, this should be done.